specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.6/TextAnalytics.json

```diff
@@ -0,0 +1,3055 @@
+          "ZAIdentificationNumber",
+          "KRResidentRegistrationNumber",
+          "ESDNI",
+          "ESSocialSecurityNumber",
+          "ESTaxIdentificationNumber",
+          "SQLServerConnectionString",
+          "SENationalID",
+          "SENationalIDV2",
+          "SEPassportNumber",
+          "SETaxIdentificationNumber",
+          "SWIFTCode",
+          "CHSocialSecurityNumber",
+          "TWNationalID",
+          "TWPassportNumber",
+          "TWResidentCertificate",
+          "THPopulationIdentificationCode",
+          "TRNationalIdentificationNumber",
+          "UKDriversLicenseNumber",
+          "UKElectoralRollNumber",
+          "UKNationalHealthNumber",
+          "UKNationalInsuranceNumber",
+          "UKUniqueTaxpayerNumber",
+          "USUKPassportNumber",
+          "USBankAccountNumber",
+          "USDriversLicenseNumber",
+          "USIndividualTaxpayerIdentification",
+          "USSocialSecurityNumber",
+          "UAPassportNumberDomestic",
+          "UAPassportNumberInternational",
+          "Organization",
+          "Email",
+          "URL",
+          "Age",
+          "PhoneNumber",
+          "IPAddress",
+          "Date",
+          "Person",
+          "Address",
+          "All",
+          "Default"
+        ]
+      },
+      "type": "array",
+      "uniqueItems": true
+    },
+    "EntityResolutions": {
+      "properties": {
+        "dateTime.TimeRange": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeRangeResolution"
+          }
+        },
+        "dateTime.Time": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeResolution"
+          }
+        },
+        "quantity.Dimension": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DimensionResolution"
+          }
+        }
+      }
+    },
+    "TimeRangeResolution": {
+      "type": "object",
+      "properties": {
+        "timex": {
+          "type": "string",
+          "description": "timex expression representation"
+        },
+        "start": {
+          "type": "string",
+          "description": "beginning date/time of the interval"
+        },
+        "end": {
+          "type": "string",
+          "description": "ending date/time of the interval"
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "The probability of the resolution instance being correct for the extracted text."
+        }
+      }
+    },
+    "TimeResolution": {
+      "type": "object",
+      "properties": {
+        "timex": {
+          "type": "string",
+          "description": "timex expression representation"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "time"
+          ],
+          "description": "The resolution type"
+        },
+        "value": {
```

Comment:
Isn't `value` already captured in the `Entity` object itself in the `text` property?  If so, maybe we can just remove `value` from these objects and you can ignore my other comment above in the `TimeResolution` object.

Reply:
No, they are differ. The "Text" property represents a substring from the original query. Here, the value denotes a semantically equivalent interpretation over the "Text" that is more digestible in a programming language. For instance your query might be: "I have two apples". The "Text" property will be "two", but the "value" in the resolution will be "2" in integer.

Reply:
Oh ok, I see, that makes sense now.